polynote-frontend/polynote/ui/component/about.ts

```diff
@@ -185,12 +185,14 @@ export class About extends FullScreenModal {
                         ]);
                         const actionsEl = div([], [
                             iconButton(['start'], 'Start kernel', 'power-off', 'Start').click(() => {
-                                this.publish(new KernelCommand(path, 'kill'));
+                                this.publish(new KernelCommand(path, 'start'));
```

Comment:
🤦‍♂

Reply:
Guessing this is my doing 😛

Reply:
Nah this was all me: https://github.com/polynote/polynote/pull/485/files#diff-36c99d07368a1579806b1d11c5d9f8ed 😅